### ISSUE TITLE
Complete forum installation with exported DB

### ISSUE THREAD

**Issue by ghost** [toger5/godotdevs-Forum#3]
Right now, there are only the smileys and the custom theme in the git repo. One has to download the same version of SMF (easy) and make the initial installation of the forum software (not so easy to end up with a system that is exactly the same). One has to download and activate the same set of packages to match the system that is supposed to run the custom theme.

Maybe we could upload the whole forum as it is now installed on the live system. Complete with the directories `Sources` and `Packages`, including some files in `avatars` and `attachments` so that there is some content to test the forum with. The directory `cache` should be empty.

At the root of the Git repo should be a text file with the database in form of SQL statements. (SQL export) It should just contain the structure of the board and some postings with attachments and karma, so that everything can be tested. There are two ways to get such a file:
1. Export the live database and strip anything unneeded: Any personal messages and their attachments, accounts and their passwords and postings after a certain date (it would be sufficient to have the postings before the announcement).
2. Make a fresh local installation, recreate the board structure and create some standard test accounts and some postings with karma.

I think it's easier to go with a copy of the live system. I can do the "cleaning" of the live database export.

With a complete install in the repo and the sample database, it is possible to just clone the repo, execute the database import and be done. Now everyone is having an identical system to develop features on. Changes to files that are done with the forum software itself (package manager for example) are still doable. If a package is desirable, it should be installed and committed like normal, except that a new database export has to be done if the package touches the database.

What do you think?

**Comment by toger5** [comment]
Yes maybe that is what we should do.
I had this setup before but I didn't want to use it anymore because I thought the db shouldn't be online and It is less confusion without the whole forum.
(but i forgot packages)
Maybe I will do that later on. But currently I have no time.
But I think you got your local version and Kiori sün and I also do. So for now we are fine I guess.

**Comment by ghost** [comment]
You are right, the live db shouldn't be online. If you want, you can just send me the zipped up root of the forum and the db export. I don't know how you can access the db on the hosted server (maybe phpMyAdmin?), but it should be easy to get a export of the db. I would take care of stripping the db off private data.

But we are mostly fine for now, that's right. It's not high priority. :)

**Comment by toger5** [comment]
Yes configuring the db is very easy (phpMyAdmin and some sql...) so that should be no problem.
And we may consider uploading the whole folder on git.
So this might be the approach we will take sooner or later It depends how many people are willing to help ;)
